rally-domain/src/main/groovy/yakworks/rally/orgs/model/PartitionOrg.groovy

```diff
@@ -0,0 +1,26 @@
+/*
+* Copyright 2024 Yak.Works - Licensed under the Apache License, Version 2.0 (the "License")
+* You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+*/
+package yakworks.rally.orgs.model
+
+import gorm.tools.model.NameNum
+import gorm.tools.repository.model.RepoEntity
+import grails.compiler.GrailsCompileStatic
+import grails.persistence.Entity
+import yakworks.security.audit.AuditStamp
+
+@Entity
+@AuditStamp
+@GrailsCompileStatic
+class PartitionOrg implements NameNum, RepoEntity<PartitionOrg>, Serializable {
```

Comment:
We have Client and Company too, and now PartitionOrg. We do them all different. Lets bring some consistency to it. Do we need Company? @jdabal that used at CED?

Reply:
do we need a repo to verify here? Whats the strategy going to be for rest on this? none? Will UI be editing the PartitionOrg or only the Org which will update this?

Reply:
CED is using Company (main and badDebt) and PartitionOrg (credit iffices). 
No one is using Client, even if there is one company we use company. 
So I would keep Company and PartitionOrg